config/environments/production.rb

```diff
@@ -31,7 +31,7 @@
   # config.action_dispatch.x_sendfile_header = "X-Accel-Redirect" # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = ENV["ACTIVE_STORAGE_SERVICE"].to_sym || :local
```

Comment:
I _think_ this will fail entirely if `ACTIVE_STORAGE_SERVICE` is not defined.  Probably better to use `ENV.fetch`